target/linux/ramips/dts/mt7620a_netgear_ex6120.dts

```diff
@@ -0,0 +1,58 @@
+/* This file is released into the public domain */
+
+/dts-v1/;
+
+#include "mt7620a_netgear_ex3700_ex6130.dtsi"
+
+/ {
+	compatible = "netgear,ex6120", "ralink,mt7620a-soc";
+	model = "Netgear EX6120";
+
+	aliases {
+		led-boot = &led_power_green;
+		led-failsafe = &led_power_green;
+		led-running = &led_power_green;
+		led-upgrade = &led_power_green;
+		label-mac-device = &ethernet;
```

Comment:
Did you reevaluate the MAC address assignment on your device?

Reply:
No, and I have no idea what to do or what this even means. Maybe @playya has done this when creating the code.

Reply:
This means that the MAC address on the label of your device corresponds to the address of the devices LAN port.

Just take the notes you made when checking that the OEM MAC address assignment of your device matches the setup of EX6130, and check whether the address for the ethernet port matches the address on the devices label.

Reply:
I do not have certainty in the labels due to multiple devices having been repaired, which is where the lid with the sticker comes off...

Reply:
Then just drop the label-mac-device line if you are not sure about it.